modules/internal/ChapelBase.chpl

```diff
@@ -1032,20 +1032,23 @@ module ChapelBase {
     return if x != 0i then true else false;
 
   pragma "dont disable remote value forwarding"
+  pragma "no copy return"
```

Comment:
It's OK to leave it for now, but I'd expect this function to need neither "no copy return" nor "no auto destroy".